attention_and_transformers.ipynb

```diff
@@ -0,0 +1,4397 @@
+{
```

Comment:
<p>[No action needed] Just finished reviewing this, and just wanted to say this colab looks amazing!! I'm sure the practice will be a success :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2022/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='236'/>